docs/docs/cmd/spo/navigation/navigation-node-list.md

```diff
@@ -49,14 +55,36 @@ m365 spo navigation node list --webUrl https://contoso.sharepoint.com/sites/team
 === "Text"
```

Comment:
Could we use the current options as defaultProperties output, so the Text and CSV output doesn't change?